docker/.env.develop

```diff
@@ -0,0 +1,3 @@
+##Cloud
+#Google Cloud
+IP=93.38.115.75
```

Comment:
Are you planning to encrypt this file when we have  keys for the bucket cloud (Google, AWS, etc ...) ?